static/docs/user-guide/contributing/core.md

```diff
@@ -250,6 +251,25 @@ $ export AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountN
 
 <details>
 
+### Click for Google Drive testing instructions
+
+#### WARNING: Do not share Google Drive access token with anyone to avoid unauthorized usage of your Google Drive.
```

Comment:
I think we don't need `####` here. Just make put ❗instead of `#### WARNING`

Reply:
I don't think we have ❗ anywhere actually 😋 except in .github/PULL_REQUEST_TEMPLATE.md

What we use a lot is a not starting with **Note!**:

```md
> **Note!** Some text
> wrapped appropriately...
```

Reply:
But I see in https://github.com/iterative/dvc.org/pull/833#pullrequestreview-327104779 you don't want this as a note. Maybe no special symbol, just a paragraph or some bold text?

Reply:
I don't have a strong opinion here, ❗ is fine. I just wanted to note we don't actually use this anywhere else in docs. (Perhaps we should do so more?)

Reply:
I think it's good to start using those emojis - ❗and 💡. They are visible and easy to understand. When you use bold text it's not that clear what is the meaning of the Note. And **Note!** is just cumbersome a bit to me. But I'm fine with either to be honest.

Reply:
❗seems attracts more attention for the required action. Do we fine to go with it?

Reply:
Yep. I just haven't had time to finish my review, sorry. I was at a conference and I'm traveling from it today but trying to do this first thing when I have a chance.